graph/articles/coreTypes.md

```diff
@@ -0,0 +1,97 @@
+# Core Types
+
+## Overview
+
+Types exist in graph which are highly-connected/central to the graph ecosystem. Often, these types are in the position of containing many structural properties relevant to other APIs, because they are connected to many entities in the graph.
+
+Structural properties should be only added to these core types when they are properties of the entity itself and strictly not for the purpose of convenience due to the entity's position in the graph.
+
+## Core Types in Graph
+
+The following types are identified as core types, and will require strong justification to allow new structural properties to be added in all cases.
+
+- ```user```
+- ```group```
+- ```device```
+
+## Alternatives to Adding Structural Properties
+
+Instead of adding a structural property to the existing type (`user`, `group` or `device`), create a new type that models the information captured in the proposed structural property(s).
```

Comment:
So we are kind of mixing core types and existing types here, which starts to get confusing.  We should stick to one.
Suggest changing existing -> core (or maybe "existing core type").